tests/integrate/217_NO_out_pot/refSPIN1_POT.cube

```diff
@@ -1,2312 +1,296 @@
- 1.706e-01 1.706e-01 1.588e-01 1.418e-01 1.243e-01 6.678e-02
- -1.932e-02 -1.270e-01 -2.592e-01 -3.828e-01 -4.743e-01 -5.541e-01
- -5.909e-01 -5.541e-01 -4.743e-01 -3.828e-01 -2.592e-01 -1.270e-01
- -1.932e-02 9.690e-02 1.765e-01 1.939e-01 2.182e-01 2.426e-01
- 2.455e-01 2.426e-01 2.182e-01 1.939e-01 1.765e-01 9.690e-02
- -6.017e-02 -1.630e-01 -2.828e-01 -3.818e-01 -4.426e-01 -4.895e-01
- -4.895e-01 -4.426e-01 -3.818e-01 -2.828e-01 -1.630e-01 -6.017e-02
- 6.678e-02 1.765e-01 2.168e-01 2.492e-01 2.759e-01 2.740e-01
- 2.740e-01 2.759e-01 2.492e-01 2.168e-01 1.765e-01 6.678e-02
- -1.223e-01 -2.248e-01 -3.341e-01 -4.091e-01 -4.562e-01 -4.792e-01
- -4.562e-01 -4.091e-01 -3.341e-01 -2.248e-01 -1.223e-01 -4.236e-04
- 1.243e-01 1.939e-01 2.492e-01 2.807e-01 2.639e-01 2.511e-01
- 2.639e-01 2.807e-01 2.492e-01 1.939e-01 1.243e-01 -4.236e-04
- -2.345e-01 -3.486e-01 -4.426e-01 -5.068e-01 -5.471e-01 -5.471e-01
- -5.068e-01 -4.426e-01 -3.486e-01 -2.345e-01 -1.034e-01 4.178e-02
- 1.418e-01 2.182e-01 2.759e-01 2.639e-01 2.375e-01 2.375e-01
- 2.639e-01 2.759e-01 2.182e-01 1.418e-01 4.178e-02 -1.034e-01
- -4.150e-01 -5.541e-01 -6.697e-01 -7.580e-01 -7.911e-01 -7.580e-01
- -6.697e-01 -5.541e-01 -4.150e-01 -2.557e-01 -8.842e-02 5.784e-02
- 1.588e-01 2.426e-01 2.740e-01 2.511e-01 2.375e-01 2.511e-01
- 2.740e-01 2.426e-01 1.588e-01 5.784e-02 -8.842e-02 -2.557e-01
- -7.148e-01 -9.421e-01 -1.133e+00 -1.232e+00 -1.232e+00 -1.133e+00
- -9.421e-01 -7.148e-01 -4.748e-01 -2.605e-01 -7.564e-02 7.361e-02
- 1.706e-01 2.455e-01 2.740e-01 2.639e-01 2.639e-01 2.740e-01
- 2.455e-01 1.706e-01 7.361e-02 -7.564e-02 -2.605e-01 -4.748e-01
- -1.221e+00 -1.557e+00 -1.796e+00 -1.871e+00 -1.796e+00 -1.557e+00
- -1.221e+00 -8.604e-01 -5.222e-01 -2.675e-01 -6.255e-02 8.272e-02
- 1.706e-01 2.426e-01 2.759e-01 2.807e-01 2.759e-01 2.426e-01
- 1.706e-01 8.272e-02 -6.255e-02 -2.675e-01 -5.222e-01 -8.604e-01
- -1.972e+00 -2.653e+00 -3.165e+00 -3.165e+00 -2.653e+00 -1.972e+00
- -1.427e+00 -9.612e-01 -5.606e-01 -2.733e-01 -6.255e-02 7.361e-02
- 1.588e-01 2.182e-01 2.492e-01 2.492e-01 2.182e-01 1.588e-01
- 7.361e-02 -6.255e-02 -2.733e-01 -5.606e-01 -9.612e-01 -1.427e+00
- -3.824e+00 -5.667e+00 -6.524e+00 -5.667e+00 -3.824e+00 -2.333e+00
- -1.543e+00 -1.001e+00 -5.606e-01 -2.675e-01 -7.564e-02 5.784e-02
- 1.418e-01 1.939e-01 2.168e-01 1.939e-01 1.418e-01 5.784e-02
- -7.564e-02 -2.675e-01 -5.606e-01 -1.001e+00 -1.543e+00 -2.333e+00
- -8.272e+00 -1.115e+01 -1.115e+01 -8.272e+00 -4.705e+00 -2.492e+00
- -1.543e+00 -9.612e-01 -5.222e-01 -2.605e-01 -8.842e-02 4.178e-02
- 1.243e-01 1.765e-01 1.765e-01 1.243e-01 4.178e-02 -8.842e-02
- -2.605e-01 -5.222e-01 -9.612e-01 -1.543e+00 -2.492e+00 -4.705e+00
- -1.425e+01 -1.631e+01 -1.425e+01 -9.341e+00 -4.705e+00 -2.333e+00
- -1.427e+00 -8.604e-01 -4.748e-01 -2.557e-01 -1.034e-01 -4.236e-04
- 6.678e-02 9.690e-02 6.678e-02 -4.236e-04 -1.034e-01 -2.557e-01
- -4.748e-01 -8.604e-01 -1.427e+00 -2.333e+00 -4.705e+00 -9.341e+00
```

Comment:
Is it possible to discard the 2312 lines of data, because there is no need to record these numbers. It's a waste to record a large number of data in Github history.

Reply:
Yes, as can be seen that 2312 lines of data (in red) have been replaced with 296 lines of data (in green).